src/awesome/client.rs

```diff
@@ -55,11 +107,51 @@ fn method_setup<'lua>(lua: &'lua Lua,
     // TODO Do properly
     use super::dummy;
     builder.method("connect_signal".into(), lua.create_function(dummy)?)?
-           .method("get".into(), lua.create_function(dummy_table)?)
+           .method("get".into(), lua.create_function(get_clients)?)
 }
 
 impl_objectable!(Client, ClientState);
 
-fn dummy_table<'lua>(lua: &'lua Lua, _: rlua::Value) -> rlua::Result<Table<'lua>> {
-    Ok(lua.create_table()?)
+fn get_clients<'lua>(lua: &'lua Lua, _: rlua::Value) -> rlua::Result<Table<'lua>> {
+    let clients: Vec<Client> = lua.named_registry_value::<Vec<AnyUserData>>(CLIENTS_HANDLE)?
+                                      .into_iter()
+                                      .map(|obj| Client::cast(obj.into()).unwrap())
+                                      .collect();
+    let table = lua.create_table()?;
+
+    for (i, client) in clients.iter().enumerate() {
```

Comment:
rlua implement `ToLua` for `Vec<T>` [see here](https://docs.rs/rlua/0.13.0/rlua/prelude/trait.ToLua.html#impl-ToLua%3C'lua%3E-15). It will automatically convert it into what your  doing here manually.

So you can just do 

```rust
let client: Vec<Client> = ...;
client.to_lua(lua)
```